pkg/app/validate_app.go

```diff
@@ -63,8 +62,9 @@ func NewValidator(config ValidatorConfig) (*Validator, error) {
 			K8sClient: config.K8sClient.K8sClient(),
 			Logger:    config.Logger,
 
-			ProjectName: project.Name(),
-			Provider:    config.Provider,
+			ProjectName:            project.Name(),
+			Provider:               config.Provider,
+			ValidateResourcesExist: true,
```

Comment:
Should we make this part of the Config, so that it is customizable?

Reply:
No its fine to be hardcoded but it would be good to have a comment explaining this.

In app-operator we will need to set it to false.

Reply:
Noted! Thanks